src/test/java/org/prebid/server/bidder/adrino/AdrinoBidderTest.java

```diff
@@ -0,0 +1,299 @@
+package org.prebid.server.bidder.adrino;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Device;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.request.Native;
+import com.iab.openrtb.request.Publisher;
+import com.iab.openrtb.request.Site;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import io.netty.handler.codec.http.HttpHeaderValues;
+import io.vertx.core.http.HttpMethod;
+import org.junit.Before;
+import org.junit.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.adrino.ExtImpAdrino;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.proto.openrtb.ext.response.ExtBidPrebid;
+import org.prebid.server.util.HttpUtil;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+
+public class AdrinoBidderTest extends VertxTest {
+
+    private static final String ENDPOINT_URL = "https://prd-prebid-bidder.adrino.io/openrtb/bid";
+
+    private AdrinoBidder adrinoBidder;
+
+    @Before
+    public void setUp() {
+        adrinoBidder = new AdrinoBidder(ENDPOINT_URL, jacksonMapper);
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnHttpRequestWithCorrectBodyHeadersAndMethod() {
+        // given
+        // Build the Imp object
+        Imp imp = Imp.builder()
+                .xNative(Native.builder().request("{json string}").ver("1.2").build())
+                .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpAdrino.of("123456")))).build();
+
+        // Build the Site object
+        Site site = Site.builder()
+                .publisher(Publisher.builder().id("1").build())
+                .page("some-page-url")
+                .build();
+
+        // Build the Device object
+        Device device = Device.builder()
+                .w(1920)
+                .h(800)
+                .build();
+
+        // Build the BidRequest object
+        BidRequest bidRequest = BidRequest.builder()
+                .id("test-request-id")
+                .imp(singletonList(imp))
+                .site(site)
+                .device(device)
+                .build();
```

Comment:
I don't think we need such a complex `BidRequest` for a bidder that just encode it without doing anything. Can be simplified to
```
        final BidRequest bidRequest = BidRequest.builder().id("test-request-id").build();
```